sqlmesh/core/model/kind.py

```diff
@@ -94,6 +99,17 @@ def model_kind_name(self) -> ModelKindName:
         return self
 
 
+def _unique_key_validator(v: t.Any) -> t.List[str]:
+    if isinstance(v, exp.Identifier):
+        return [v.this]
```

Comment:
this isn't guaranteed to run a string for tuple, 

maybe change all this to .name